lib/libcc1/Levelset.cpp

```diff
@@ -249,7 +249,9 @@ long ccl::LevelData::read(ccl::Stream* stream, bool forClipboard)
     m_chips = stream->read16();
     dataSize -= 3 * sizeof(unsigned short);
 
-    if (stream->read16() != 1)
+    long mapDetail = stream->read16();
```

Comment:
It looks like this field is passed to the RLE decompression function, but isn't actually used by it...  Based on other notes from the wiki, this might be an (unused) compression type.  I'd recommend renaming it as such (and changing the type to match what's read):

```suggestion
    uint16_t compressionType = stream->read16();
```

Reply:
Fair enough!